webserver_tests.py

```diff
@@ -0,0 +1,272 @@
+import unittest
+from block import Block
+from get_block_details import shorten_rc_title, get_rc_values_for_block, get_rc_values_for_block_from_object, set_rc_values_for_block_from_pvs, set_rc_values_for_blocks
+
+
+class TestWebServer(unittest.TestCase):
+    def test_shorten_rc_title_for_default_case(self):
+        #Arrange
+        test_pv = "TE:NDLT910:DAE:COUNTRATE.VAL"
+
+        #Act
+        shortened_title = shorten_rc_title(test_pv)
+
+        #Assert
+        self.assertEquals(shortened_title, "COUNTRATE.VAL")
+
+    def test_new_block_rc_in_range(self):
+        #Arrange
+        test_pv = "TE:NDLT910:CS:SB:NEW_BLOCK:RC:INRANGE.VAL"
+
+        #Act
+        shortened_title = shorten_rc_title(test_pv)
+
+        #Assert
+        self.assertEquals(shortened_title, "NEW_BLOCK:RC:INRANGE.VAL")
+
+    def test_new_block_rc_enabled(self):
+        # Arrange
+        test_pv = "TE:NDLT910:CS:SB:NEW_BLOCK:RC:ENABLED.VAL"
+
+        # Act
+        shortened_title = shorten_rc_title(test_pv)
+
+        # Assert
+        self.assertEquals(shortened_title, "NEW_BLOCK:RC:ENABLED.VAL")
+
+    def test_old_block_rc_in_range(self):
+        # Arrange
+        test_pv = "TE:NDLT910:CS:SB:OLD_BLOCK:RC:INRANGE.VAL"
+
+        # Act
+        shortened_title = shorten_rc_title(test_pv)
+
+        # Assert
+        self.assertEquals(shortened_title, "OLD_BLOCK:RC:INRANGE.VAL")
+
+    def test_random_block_rc_in_range(self):
+        # Arrange
+        test_pv = "TE:NDLT910:CS:SB:ANY_BLOCK:RC:INRANGE.VAL"
+
+        # Act
+        shortened_title = shorten_rc_title(test_pv)
+
+        # Assert
+        self.assertEquals(shortened_title, "ANY_BLOCK:RC:INRANGE.VAL")
+
+    def test_rc_in_pv_doesnt_count(self):
+        # Arrange
+        test_pv = "TE:NDLT910:RC.VAL"
+
+        # Act
+        shortened_title = shorten_rc_title(test_pv)
+
+        # Assert
+        self.assertEquals(shortened_title, "RC.VAL")
+
+    def test_larmor_block_high_rc(self):
+        # Arrange
+        test_pv = "IN:LARMOR:CS:SB:CJHCent:RC:HIGH.VAL"
+
+        # Act
+        shortened_title = shorten_rc_title(test_pv)
+
+        # Assert
+        self.assertEquals(shortened_title, "CJHCent:RC:HIGH.VAL")
+
+    def test_larmor_block_low_rc(self):
+        # Arrange
+        test_pv = "IN:LARMOR:CS:SB:Chi:RC:LOW.VAL"
+
+        # Act
+        shortened_title = shorten_rc_title(test_pv)
+
+        # Assert
+        self.assertEquals(shortened_title, "Chi:RC:LOW.VAL")
+
+    def test_get_one_rc_value_for_block(self):
+        # Arrange
+        test_block = "NEW_BLOCK"
+        test_pvs = {"NEW_BLOCK:RC:LOW.VAL" : 10, "MORGAN:RC:LOW.VAL" : 100,  "OLD_BLOCK:RC:LOW.VAL" : 7}
+
+        # Act
+        rc_values = get_rc_values_for_block(test_block, test_pvs)
+
+        # Assert
+        self.assertEquals(rc_values, 10)
+
+    def test_get_one_rc_value_for_block(self):
```

Comment:
This is a redefinition of the test above, which means:
 - This test does not run
 - This test actually fails (see comment [above](https://github.com/ISISComputingGroup/JSON_bourne/pull/14/files#diff-516f5df5fa600fbbd033e61c35d81f94R81)) but when you run the tests they will not complain.